pkg/options/types.go

```diff
@@ -105,14 +106,52 @@ func (r *ResourceSet) Type() string {
 }
 
 // NodeType represents a nodeName to query from.
-type NodeType string
+type NodeType map[string]struct{}
+
+func (n *NodeType) Set(value string) error {
+	s := *n
```

Comment:
Make `s` a new map.

Because if `n` has 10 node names with 2 duplicated.  Length of `cols := make([]string, 0, len(n))` is 10 instead of 9.

Reply:
> Make `s` a new map.
> 
> Because if `n` has 10 node names with 2 duplicated. Length of `cols := make([]string, 0, len(n))` is 10 instead of 9.

Is it possible to configure the function NewOptions？
```
// NewOptions returns a new instance of `Options`.
func NewOptions() *Options {
	return &Options{
		Resources:            ResourceSet{},
		MetricAllowlist:      MetricSet{},
		MetricDenylist:       MetricSet{},
		MetricOptInList:      MetricSet{},
		Node:                 NodeType{},
		AnnotationsAllowList: LabelsAllowList{},
		LabelsAllowList:      LabelsAllowList{},
	}
}
```